docs/extensions/developer.rst

```diff
@@ -0,0 +1,53 @@
+
+Developer Tools
+===============
+
+The following modules provide support to both core and extension Tripal developers.
+
+Tripal Docker (Centos)
```

Comment:
tehse are deprecated.  i made an issue about it in the module.  I'm going to replace it with our tripaldock tool.